packages/theming-react-native/src/index.ts

```diff
@@ -3,6 +3,6 @@ export * from './Theme.types';
 export { IThemeLayerProps, ThemeLayer } from './ThemeLayer';
 export * from './platform';
 export * from './ThemeProvider';
-export * from './ThemeProvider.types';
 export * from './ThemeContext';
 export * from './NativeModule';
+export * from '@uifabricshared/themed-stylesheet';
```

Comment:
Does this actually need to export everything from themed-stylesheet?

Reply:
It's a good question.  Early feedback with some consumers of @theming-react-native bemoaned having to import from a second package and I thought we were leaning toward having theming-react-native being a "friendly" reexport for the theming concepts in the repo--- I think the theming ramp is effectively already reexported from here.